<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="voices">
      <property name="title">
       <string>Voices</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="voice1">
         <property name="title">
          <string>Voice 1</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <widget class="QDial" name="levelKnob_V1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="levelButton_V1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>select voice</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="levelLabel_V1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="fineCoarseButton_V1">
            <property name="text">
             <string>toggle fine/coarse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDial" name="fineCoarseKnob_V1"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="mute_V1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mute</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="fineCoarseLabel_V1">
            <property name="text">
             <string>FINE/COARSE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="display_V1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>.....</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QGroupBox" name="voice3">
         <property name="title">
          <string>Voice 3</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QDial" name="fineCoarseKnob_V3"/>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="fineCoarseButton_V3">
            <property name="text">
             <string>toggle fine/coarse</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="levelLabel_V3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDial" name="levelKnob_V3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="levelButton_V3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>select voice</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="fineCoarseLabel_V3">
            <property name="text">
             <string>FINE/COARSE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="mute_V3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mute</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="display_V3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>.....</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="voice2">
         <property name="title">
          <string>Voice 2</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="levelLabel_V2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="fineCoarseButton_V2">
            <property name="text">
             <string>toggle fine/coarse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDial" name="fineCoarseKnob_V2"/>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="display_V2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>.....</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="levelButton_V2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>select voice</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDial" name="levelKnob_V2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="fineCoarseLabel_V2">
            <property name="text">
             <string>FINE/COARSE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="mute_V2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mute</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="voice4">
         <property name="title">
          <string>Voice 4</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="2">
           <widget class="QDial" name="fineCoarseKnob_V4"/>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="fineCoarseButton_V4">
            <property name="text">
             <string>toggle fine/coarse</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="levelLabel_V4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>LEVEL</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDial" name="levelKnob_V4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="levelButton_V4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>select voice</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="fineCoarseLabel_V4">
            <property name="text">
             <string>FINE/COARSE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="mute_V4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>mute</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="display_V4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>.....</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
